UnityProject/Assets/Scripts/Items/Implants/Xenomorphs/XenomorphLarvae.cs

```diff
@@ -15,8 +15,8 @@ namespace HealthV2
 	public class XenomorphLarvae : BodyPartFunctionality
 	{
 		[SerializeField]
-		[Tooltip("This GameObject will be spawned from the Larvae when its time")]
-		private GameObject SpawnedLarvae;
+		[Tooltip("This Character will be spawned from the Larvae when its time")]
+		private Occupation XenomorphLarvaeOccupation;
```

Comment:
why change from a prefab to spawn to an occupation?

Reply:
it doesn't take a prefab as a spawn parameter